ansible/roles/rancher/tasks/rancher.yml

```
@@ -4,5 +4,16 @@
   when: "'rancher-stable' not in repos.stdout"
 
 - name: Install Rancher
-  command: "helm install rancher-stable/rancher --name rancher --namespace cattle-system --version {{ rancher_version }} --set tls=external --set privateCA=true --set addLocal=false --set auditLog.level=1 --description='RanchHand Deploy' --wait"
-  when: "'rancher' not in charts.stdout"
\ No newline at end of file
+  command: "helm install rancher-stable/rancher -f values.yml
```

You'll need to use proper YAML support for multi-line strings

```suggestion
  command: <-
    helm install rancher-stable/rancher -f values.yml
```